test/alias.lean

```diff
@@ -0,0 +1,95 @@
+import Std.Tactic.Alias
+import Std.Tactic.GuardMsgs
+
+set_option linter.unusedVariables false
+
+open Lean Meta
+namespace A
+
+/-- doc string for foo -/
+theorem foo : 1 + 1 = 2 := rfl
+
+/-- doc string for `alias foo` -/
+alias foo1 := foo
```

Comment:
hovering `foo1` does not show the type, unlike in `def foo1 := 1`. I think you are missing a `ofTermInfo` info tree node placed over the declaration span

Reply:
Ah! That's what was missing! Thanks!

Reply:
I just realized that I don't know the proper way to do this. Any pointers?

Reply:
Found it! Weird day... it turns out it was at the *first* place I looked but somehow completely missed it. 🤣